packages/manager/apps/web/client/app/hosting/offer/upgrade/hosting-offer-upgrade.controller.js

```diff
@@ -55,7 +57,10 @@ angular.module('App').controller(
         downgradeAgree: false,
       };
 
-      this.WucUser.getUser().then((user) => {
+      const user = this.coreConfig.getUser();
+      this.Hosting.getCatalog(user.ovhSubsidiary).then((catalog) => {
+        this.user = user;
+        this.catalog = catalog;
```

Comment:
**nitpick:** Temporary local variable can be avoided.
```suggestion
      this.user = this.coreConfig.getUser();
      this.Hosting.getCatalog(this.user.ovhSubsidiary).then((catalog) => {
        this.catalog = catalog;
```

Reply:
it avoid to accessing lot times to user object.
It used 3 times after declaration.